geopandas/base.py

```diff
@@ -183,11 +183,10 @@ def area(self):
     def crs(self):
         """The Coordinate Reference System (CRS) as a ``pyproj.CRS`` object.
 
-        Returns None if the CRS is not set, and to set the value it
-        :getter: Returns a ``pyproj.CRS`` or None. When setting, the value
-        can be anything accepted by
-        :meth:`pyproj.CRS.from_user_input() <pyproj.crs.CRS.from_user_input>`,
-        such as an authority string (eg "EPSG:4326") or a WKT string.
```

Comment:
Removing a note about setting as this will be disabled and users will be forced to use `set_crs` instead.

Reply:
Actually, in https://github.com/geopandas/geopandas/pull/3088 this was only deprecated for overriding a CRS, not setting a CRS from none. We could choose to change that though.

Reply:
Hmm, I would probably vote for consistency here. Meaning requiring `set_crs` in all cases.